File: Vehiculo.java

package parking;

public class Vehiculo {
    private String modelo;
    
    private String placa;
    private String TipoVehiculo; //Carro o Moto
    private Usuario dueño;

    public Vehiculo(String modelo, String placa, String TipoVehiculo, Usuario dueño) {
        this.modelo = modelo;
        this.placa = placa;
        this.TipoVehiculo = TipoVehiculo;
        this.dueño = dueño;
    }

    public Usuario getDueño(){
        return dueño;
    }
    
}

File: ProyectosC#/class/Vehiculo.cs
public class Vehiculo
{
    public Usuario Dueno { get; set; }
    public string Marca { get; set; }
    public string Modelo { get; set; }
    public string TipoVehiculo { get; set; }  // "carro" o "moto"

    public Vehiculo(Usuario dueno, string marca, string modelo, string tipoVehiculo)
    {
        Dueno = dueno;
        Marca = marca;
        Modelo = modelo;
        TipoVehiculo = tipoVehiculo;
    }

    public Usuario GetDueno()
    {
        return Dueno;
    }
}
